WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/logic/interval/IntervalManager.java

```diff
@@ -10,8 +10,8 @@
 public class IntervalManager extends IDEIntervalManagerBase {
```

Comment:
Not sure, do we really need separate classes for IntervalManager and EventManager (i.e. isn't one super class enough)? The EC and IJ implementations look similar, too.